tests/test_service.py

```diff
@@ -1,6 +1,55 @@
-"""Tests go here."""
+import threading, time
+import pytest
+import os
+from producer import publish_message
+from kafka import KafkaConsumer
 
+msg = 'Aiops-testing'
 
-def test_something():
-    """Test something."""
-    pass
+class Consumer(threading.Thread):
+    def __init__(self, server, topic, msg=None):
+        threading.Thread.__init__(self)
+        self.stop_event = threading.Event()
+        self.server = server
+        self.topic = topic
+        self.msg = msg
+
+    def stop(self):
+        self.stop_event.set()
+
+    def run(self):
+        consumer = KafkaConsumer(self.topic, bootstrap_servers=self.server, request_timeout_ms=10)
+
+        while not self.stop_event.is_set():
+            if consumer.poll():
+                message = consumer.poll()
+                if self._check_message(message):
+                    break
+
+    def _check_message(self, msg=None):
+        if self.msg in msg:
```

Comment:
This will raise `TypeError: argument of type 'NoneType' is not iterable` if `msg` is the default of `None`

Also, I'd prefer ensuring the message content with a more strict check like `self.msg == msg`